docs/gambit/mutation-verifier.md

```diff
@@ -13,25 +13,31 @@ In the rest of the document,
 
 ## Installations and Setup
 
-You will require java to run the mutation testing jar.
-Installation instructions can be found [here](https://www.java.com/en/download/help/download_options.html).
+To use the mutation verifier,
+  you will need to {ref}`install the Certora Prover and its dependencies <installation>`.
+  Mutation verification currently requires the `alpha-master` version of the Certora CLI.
+To install it, run
 
-## Running the Mutation Verifier
-
-- Example:
+```sh
+pip install certora-cli-alpha-master
```

Comment:
We'll soon deploy to prod, don't we want a final version with original certora_cli?

Reply:
@shellygr we do, but when will that happen? I know some people internally were trying to use it this week for the Secureum thing so i wanted to have this up for now. Happy to update this now or once we have it ready.

Reply:
ok we decided on slack that we will merge this now and I am about to open another PR that updates this to the prod version.